docs/09-troubleshooting/common-issues.mdx

```diff
@@ -30,6 +152,42 @@ Make sure your branch is clean and all files are indeed present:
 A good way to verify this, is to (locally) clone the Unity project in a new folder and run the build
 from there.
 
+### Branch is dirty
+
+#### Error
+
+```plaintext
+Branch is dirty. Refusing to base semantic version on uncommitted changes
```

Comment:
Is this also something that can happen when LFS failed or something? Thinking of this, it might be interesting to mention a few things concerning git LFS, but that can definitely be in a different pr :)

Reply:
Yes, it's possible if you don't check out your files then your branch is dirty